### ISSUE TITLE
Freeze weights during training

### ISSUE THREAD

**Issue by BingzheWu** [apache/incubator-mxnet#4201]
Is there any solution to freeze weights? For example, I want to keep some weights unchanged when them satisfy some condition(like weight<0.5) during training.

**Comment by kevinthesun** [comment]
One solution might be using optimizer interface. You can set individual learning rate multipler for parameters. http://mxnet.io/api/python/model.html?highlight=optimizer#optimizer-api-reference

**Comment by yajiedesign** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to reopen if this is still an active issue. Thanks!
